### PR TITLE
(core) convert runningExecutionsFilter to TS

### PR DESCRIPTION
**Pull request by anotherchrisberry**
I ended up pulling a bunch of boilerplate code into a component, which seems a lot better than what's there now.

### CODE REVIEW

File: app/scripts/modules/core/cluster/cluster.service.js [hunk 1]
```diff
@@ -207,25 +207,25 @@ module.exports = angular.module('spinnaker.core.cluster.service', [
       }
 
       application.serverGroups.data.forEach(function(serverGroup) {
-        serverGroup.executions = [];
+        serverGroup.runningExecutions = [];
         executions.forEach(function (execution) {
 
-          var stages = findStagesWithServerGroupInfo(execution.stages);
+            var stages = findStagesWithServerGroupInfo(execution.stages);
```
Comment (icfantv) [inline app/scripts/modules/core/cluster/cluster.service.js]: something happened with the indents in this `forEach`.

File: app/scripts/modules/titus/serverGroup/details/serverGroupDetails.html [hunk 2]
```diff
@@ -52,43 +52,7 @@ <h3 select-on-dbl-click>
...
-          <div class="col-md-7 col-md-offset-0">
-            <span class="small"><status-glyph item="stage"></status-glyph></span> {{stage.name | robotToHuman }}
-          </div>
-          <div class="col-md-4 text-right">
-            {{stage.runningTimeInMs | duration }}
-          </div>
-        </div>
-      </div>
-    </collapsible-section>
+      <server-group-running-tasks-details server-group="serverGroup"></server-group-running-tasks-details>
```
Comment (icfantv) [inline app/scripts/modules/titus/serverGroup/details/serverGroupDetails.html]: indention is off.

File: app/scripts/modules/core/cluster/cluster.service.js (hunk 1 above)
Comment (anotherchrisberry) [inline app/scripts/modules/core/cluster/cluster.service.js]: thanks, fixed

File: app/scripts/modules/titus/serverGroup/details/serverGroupDetails.html (hunk 2 above)
Comment (anotherchrisberry) [inline app/scripts/modules/titus/serverGroup/details/serverGroupDetails.html]: yep, fixed

**Pull request merged**